woocommerce/woocommerce-android
# [Custom Amounts 4.2] UI redesign

Closes: #10230 


### Description

This Pull Request introduces a redesign of the custom amount UI, aligning it with the latest design specifications provided in Figma: https://www.figma.com/file/ichqhxDjbmZZmvJaJkGhkj/Simple-Payments-%26-Order-Creation?node-id=626%3A21620&mode=dev

### Testing instructions

1. Navigate to the Order creation screen
2. Click on "Enter custom amount" row
3. Ensure the dialog is as per new design (please note if the divider in the screen is missing, it's fixed in [this PR](https://github.com/woocommerce/woocommerce-android/pull/10298))

### Images/gif



Before                    |  After
:------------------------:|:-------------------------:
![Before](https://github.com/woocommerce/woocommerce-android/assets/1331230/bba55765-7691-4862-888e-29c36aaf846d)  |  ![After](https://github.com/woocommerce/woocommerce-android/assets/1331230/b1858534-af50-4de5-bc49-2c7e57c3f84b)
